ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecurePostalAddress.java

```diff
@@ -37,91 +37,82 @@
     public ThreeDSecurePostalAddress() {}
 
     /**
-     * @deprecated Use {@link #givenName(String)}.
+     * @deprecated Use {@link #setGivenName(String)}.
      */
     @Deprecated
-    public ThreeDSecurePostalAddress firstName(String firstName) {
-        givenName(firstName);
-        return this;
+    public void setFirstName(String firstName) {
```

Comment:
We should make sure to remove this (and any other deprecated methods), unless there's a specific reason to keep it?

Reply:
👍   These 3DS deprecated methods will be removed in #341 (but we will probably have another PR for any other remaining deprecated methods)

Reply:
Right yeah we removed these for PostalAddress but figured we would circle back and get the remaining deprecated methods for `beta3`. This PR is mainly to get the builder pattern removed since it's a big structural change.

Reply:
Sounds good!